cmd/flag_helpers.go

```diff
@@ -30,34 +31,98 @@ import (
 
 // addGetGroupsFlags adds common flags for `get groups` and `get subgroups` commands
 func addGetGroupsFlags(cmd *cobra.Command) {
-	cmd.Flags().Bool("all-available", false,
-		"Show all the groups you have access to "+
-			"(defaults to false for authenticated users, true for admin)")
-	cmd.Flags().Bool("owned", false,
-		"Limit to groups owned by the current user")
-	cmd.Flags().Bool("statistics", false,
-		"Include group statistics (admins only)")
-	cmd.Flags().String("sort", "asc",
-		"Order groups in asc or desc order. Default is asc")
-	cmd.Flags().String("search", "",
-		"Return the list of authorized groups matching the search criteria")
-	cmd.Flags().String("order-by", "name",
-		"Order groups by name or path. Default is name")
+	addAllAvailableFlag(cmd)
+	addGroupOrderByFlag(cmd)
+	addOwnedFlag(cmd)
+	addSortFlag(cmd)
+	addStatisticsFlag(cmd)
+	addSearchFlag(cmd)
+}
+
+// addGetProjectsFlags adds common flags for `get projects` commands
```

Comment:
The default sort in projects api is desc but in this function it is asc and in the flag description seems fixed for groups ?